migrations/20160912132241_create_risks_of_bias.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+exports.up = (knex) => {
+  const schema = knex.schema;
+
+  schema.createTable('risk_of_biases', (table) => {
+    table.uuid('id').primary();
+
+    table.uuid('trial_id')
+      .notNullable()
+      .references('trials.id');
+    table.text('source_id')
+      .notNullable()
+      .references('sources.id');
+    table.text('source_url')
+      .notNullable();
```

Comment:
Couldn't we receive a risk of bias assessment without a `source_url`? As when we received them as a data donation? If so, this should be `nullable()`